library/general/src/modules/Linuxrc.rb

```diff
@@ -286,6 +290,7 @@ def disable_remote(services)
     publish function: :keys, type: "list <string> ()"
     publish function: :value_for, type: "string (string)"
     publish function: :disable_remote, type: "list <string> ()"
+    publish function: :reboot_timeout, type: "string (string)"
```

Comment:
signature looks really strange, it does not have any parameter and also it returns boolean, not string.

Reply:
I must admit, I don't really know what I am doing here :) I was merely copying what I found for display_ip assuming that I want something comparable here. What I wanted to achieve is that the user can specify a parameter "reboot_timeout" with an integer value that is used in https://github.com/yast/yast-installation/pull/823 to override the default of "10" when specified. Can you suggest how I can do that?

Reply:
ok, so let me write it here. Here you should add that method does not have params and returns interger `type: "integer ()"`